CrawlerTests/LexerTests.cs

```diff
@@ -1,18 +1,28 @@
-﻿using Crawler.LexicalAnalyzer;
+﻿using System.Collections.Generic;
+using Crawler.LexicalAnalyzer;
 using Microsoft.Extensions.Logging;
 using Moq;
 using System.Linq;
+using Crawler.Configs;
+using Microsoft.Extensions.Options;
 using Xunit;
 
 namespace CrawlerTests
 {
     public class LexerTests
     {
         private readonly ILexer lexer;
+        private readonly LexerConfig config;
 
         public LexerTests()
         {
-            lexer = new Lexer(Mock.Of<ILogger<Lexer>>());
+            var options = Mock.Of<IOptions<LexerConfig>>();
```

Comment:
I'm not sure this is a good name.

Reply:
it returns an options interface, do you have another suggestion?

Reply:
You're right, but it's an interface to the LexerConfig class. Maybe configOptions? or configMock?

Reply:
Fixed